emacs.sh

```
@@ -0,0 +1 @@
+nix-shell -p emacsPackages.magit -p emacsPackages.rustic -p emacsPackages.cargo-mode -p emacsPackages.rust-mode -p emacsPackages.lsp-mode
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)